pkg/executor/importer/table_import.go

```diff
@@ -202,7 +202,7 @@ func NewTableImporter(
 		return nil, err
 	}
 
-	if err := localBackend.InitTiCIWriterGroup(ctx, getEtcdClient, e.Table.Meta(), e.DBName); err != nil {
+	if err := localBackend.InitTiCIWriterGroup(ctx, getEtcdClient, e.Table.Meta(), e.DBName, id); err != nil {
```

Comment:
@OliverS929 Can we suppose this `id` is a unique identifier for one `IMPORT INTO` job (or a `CREATE INDEX` backfilling job that the generate index of existing data)?